### PR TITLE
Fix/ Ascend Ex order book mismatched

### PR DESCRIPTION
**Pull request by Nullably**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
- Sort bids and asks on order book message
- Send pong on web socket when received ping msg

### CODE REVIEW

File: hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py [hunk 1]
```diff
@@ -75,8 +74,8 @@ async def _track_single_book(self, trading_pair: str):
                     message = await message_queue.get()
 
                 if message.type is OrderBookMessageType.DIFF:
-                    bids, asks = active_order_tracker.convert_diff_message_to_order_book_row(message)
-                    order_book.apply_diffs(bids, asks, message.update_id)
+                    # bids, asks = active_order_tracker.convert_diff_message_to_order_book_row(message)
+                    order_book.apply_diffs(message.bids, message.asks, message.update_id)
```
Comment (CrimsonJacket) [inline hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py:78]: `apply_diffs()` expects bids and asks to be `List[OrderBookRow]`. I have yet to test it myself, but I am expecting an error when accessing the `update_id` attribute in the `OrderBookRow`

File: hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py [hunk 2]
```diff
@@ -89,17 +88,14 @@ async def _track_single_book(self, trading_pair: str):
...
+                    past_diffs = list(past_diffs_window)
                     replay_position = bisect.bisect_right(past_diffs, message)
                     replay_diffs = past_diffs[replay_position:]
-                    s_bids, s_asks = active_order_tracker.convert_snapshot_message_to_order_book_row(message)
-                    order_book.apply_snapshot(s_bids, s_asks, message.update_id)
-                    for diff_message in replay_diffs:
-                        d_bids, d_asks = active_order_tracker.convert_diff_message_to_order_book_row(diff_message)
-                        order_book.apply_diffs(d_bids, d_asks, diff_message.update_id)
-
+                    order_book.apply_snapshot(message.bids, message.asks, message.update_id)
```
Comment (CrimsonJacket) [inline hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py:94]: Similar for `apply_snapshot()` it expects bids and asks to be `List[OrderBookRow]`

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
~~`apply_diffs()` and `apply_snapshot()` in `OrderBook` expects the bids and asks parameters to be `List[OrderBookRow]`. I assume this will result in an error when accessing the `update_id` attribute in the `OrderBookRow`~~

**Review by CrimsonJacket — APPROVED** [review]
Ignore my previous review. I did not catch that you've made the changes to `AscendExOrderBookMessage`

**Review by rxlxrxsx — APPROVED** [review]
Tested and confirmed fixed by QA.

**Pull request merged**